src/diff/props.js

```diff
@@ -97,6 +97,12 @@ function setProperty(dom, name, value, oldValue, isSvg) {
 		let useCapture = name !== (name = name.replace(/Capture$/, ''));
 		let nameLower = name.toLowerCase();
 		name = (nameLower in dom ? nameLower : name).slice(2);
+		
+		// Fix onInput not firing on select/radio/checkbox in Edge <= 18.
+		if (name === 'input' && (
+			type === 'select' ||
+			type === 'input' && /rad|check/.test(props.type)
```

Comment:
🤔 are `props` defined in here? Couldn't find them. Maybe I'm missing something.

Reply:
I think we need to differentiate `vnode.type` and the input type which is usually in `props.type` here. As far as I can tell both are needed to pass this check. So we'd need to pass `props` through the arguments.